### ISSUE TITLE
Password Recovery Redirect User to Sign In

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#369]
@stephanosm suggested in #204 the following improvement to the password recovery process:
> After the Password has been changed then redirect the user to the sign in page with the message Password Changed. This is one click less for the user.
![image](https://user-images.githubusercontent.com/22602396/26837106-9537e7cc-4aaa-11e7-945f-422efde51901.png)

This issue was created to document and track work on that.

**Comment by gillbalraj** [comment]
The password recovery page has been updated. 
It can be tested by clicking the **forget-password** button and then following the e-mail link to set the new password.

**Comment by joshi1983** [comment]
@stephanosm can you test this?  I heard you might be returning from a vacation in a week.

**Comment by joshi1983** [comment]
@stephanosm does this look good after setting a new password?

![image](https://user-images.githubusercontent.com/11838191/28597870-47c656b4-716e-11e7-801c-ec9b626fa383.png)

**Comment by Blandine-AA** [comment]
Stephanos is returning the 17th, not in a week:)

Dr. Blandine Fongue Lesage, PhD
AccessLocator Project Manager & Product Owner
Software Development Product Manager, Consultant
Scrum & Agile Practitioner, SFC, SMAC, SPOAC

**Comment by stephanosm** [comment]
I signed in and then changed my password. The behavior is still the same:
![image](https://user-images.githubusercontent.com/22602396/28619308-bd53317e-71d5-11e7-8a40-56c3785a608d.png)

**Comment by joshi1983** [comment]
@stephanosm you're right.  The password recovery was the only update when it should have included the password changed too.

The test instructions should include going through the password recovery process too.

**Comment by joshi1983** [comment]
@stephanosm I just noticed that if you change the password, you're not signed out.  You originally said that being redirected to the sign in page would be less clicks but isn't there fewer clicks now that you remain signed in?

This seems like a slightly different matter but do you think it is a bug that you stay signed in after changing the password?

**Comment by Blandine-AA** [comment]
@joshi1983, what do you mean you stay signed in? Is it that you got automatically signed in using the new password, or you are still signed in with the old password?

Normally, **after you validate your new password, you should be automatically signed in**. This is better! No need to go back to the sign in page.

**Comment by joshi1983** [comment]
@Blandine-AA still signed in with the old password.

How would Signing out and signing back in automatically be different to the visitor from not signing out at all?  Do you just want to go to the user profile?

**Comment by Blandine-AA** [comment]
@joshi1983 normally when you ask for a password recovery you are not signed in. So, when the password recovery process is completed, the user should be signed in by AL. No need to go through the profile.

If you are already signed in, and changed your password through the "Change Password" feature in the profile, you should still be signed in with your old password, but the next time you sign in, it will be with your new password.

**Comment by joshi1983** [comment]
@Blandine-AA that's how it works now then and this issue may be ready to close.  This is on app.accesslocator.com if you wanted to test.

When Stephanos said "I signed in and then changed my password. The behavior is still the same:", he signed in and changed his password instead of recovering a forgotten password.  That lead me to think that the change password feature also needed some changes but maybe not.

**Comment by stephanosm** [comment]
@joshi1983 and @Blandine-AA I was referring when you are sign in  and you go to your profile, choose change password you get this. I believe it should take you either back to your Profile or the Home page with the message Password Changed!
![image](https://user-images.githubusercontent.com/22602396/28883675-3fee8b68-777d-11e7-948c-2c69a0a5c594.png)

**Comment by Blandine-AA** [comment]
Sounds right. But since you requested the password change from the profile, I think the message should be on the profile page.

**Comment by joshi1983** [comment]
I unassigned myself to let other developers know that this is available for them to work on.

**Comment by passionInfinite** [comment]
@joshi1983 Assigned this issue to me!

**Comment by passionInfinite** [comment]
#549 Resolved this issue.
